scripts/run-monitor-ci-tests.bash

```diff
@@ -165,10 +165,18 @@ do
 							# download artifact
 							filename=$(basename "${path}")
 							filename="${filename::-1}" # removes extra " from end
-							curl -L -s --request GET \
-								--output "monitor-ci/test-artifacts/results/${name}/${filename}" \
-								--url "${url}" \
-								--header "Circle-Token: ${API_KEY}"
+							# put shared artifacts in the shared folder
+							if [[ *"shared"* == "${path}" ]] ; then
+							  curl -L -s --request GET \
+                  --output "monitor-ci/test-artifacts/results/shared/${name}/${filename}" \
+                  --url "${url}" \
+                  --header "Circle-Token: ${API_KEY}"
+              else
+                curl -L -s --request GET \
+                  --output "monitor-ci/test-artifacts/results/${name}/${filename}" \
+                  --url "${url}" \
+                  --header "Circle-Token: ${API_KEY}"
```

Comment:
can we reuse the curl command just use a variable for the value for the `--output` flag? Then the if/else block would just be setting the `output` variable. Might simplify this a bit!

Reply:
Good idea - done!